packages/graphql/src/classes/Node.ts

```diff
@@ -81,22 +81,22 @@ type MutableField =
     | CustomScalarField
     | CustomEnumField
     | UnionField
-    | ObjectField
+     
```

Comment:
question: Shouldn't those lines be deleted? (there are several more in other files in this PR too)

Reply:
Good spot, I think that's from where ESLint did some automatic fixing!